app/views/candidates/home/index.html.erb

```diff
@@ -29,33 +30,36 @@
 
       <p>
         To request school experience, you’ll need to provide:
-      </p>
-
-      <ul class="govuk-list govuk-list--bullet">
-        <li>personal details - including your address</li>
-        <li>contact details - including your email and telephone number</li>
-      </ul>
 
-      <p>
-        As part of making a school experience request, your details will be
-        registered with the Department for Education’s
-        <a href="https://getintoteaching.education.gov.uk/">
-          Get Into Teaching service
-        </a>.
+        By using this service, you'll be registered with the Department for
+        Education's <%= link_to 'Get Into Teaching Information Service', 'https://getintoteaching.education.gov.uk/' %>
+        to receive tailored advice about becoming a teacher.
       </p>
 
       <p>
-        This means you can get extra help and tailored advice and information
-        about becoming a teacher.
+        During school holidays, schools may take longer to respond to requests.
       </p>
 
       <p>
-        If you're registered with our service or Get Into Teaching - we'll
-        automatically add your personal and contact details to your school
-        experience request.
+        Some schools charge fees to cover their costs. These will be clearly
+        listed and need to be paid directly to the individual school.
       </p>
-
     </section>
+
+    <%- Feature.instance.from_phase(5) do %>
```

Comment:
I wasn't aware of the content of this - its feature flagged off for now so I'll approve but it'll probably need further amendments.

If the user has used the service before, they can just carry on as per normal anyway - they'll get signed in as part of the process anyway.